contracts/sol5/utils/Utils4.sol

```diff
@@ -6,7 +6,8 @@ import "../IERC20.sol";
 /**
  * @title Kyber utility file
  * mostly shared constants and rate calculation helpers
- * inherited by most of kyber contracts
+ * inherited by most of kyber contracts.
+ * previous utils implementations are for previous solidity version.
```

Comment:
versions

Reply:
but before version 5 we have one version. (for our code)